server/src/main/java/org/opensearch/action/bulk/BulkRequest.java

```diff
@@ -453,6 +479,9 @@ public void writeTo(StreamOutput out) throws IOException {
         out.writeCollection(requests, DocWriteRequest::writeDocumentRequest);
         refreshPolicy.writeTo(out);
         out.writeTimeValue(timeout);
+        if (out.getVersion().onOrAfter(MINIMAL_VERSION_SUPPORT_BATCH)) {
```

Comment:
Please use `Version.V_2_14_0` directly, there is no need to have indirection, both are constants:

```suggestion
        if (out.getVersion().onOrAfter(Version.V_2_14_0)) {
```